src/Microsoft.Health.Dicom.Core/Features/Retrieve/RetrieveResourceService.cs

```diff
@@ -123,8 +127,8 @@ public async Task<RetrieveResourceResponse> GetInstanceResourceAsync(RetrieveRes
             // transcoding of single instance
             if (needsTranscoding)
             {
-                _logger.LogInformation("Transcoding Instance");
```

Comment:
Do you think this message is useful. If checkFileSize throws an exception, we may not know what part the error is thrown

Reply:
the stack trace will show. But earlier log would tell us how many calls made to transcoding with and without eroor